packages/next/src/lib/verify-partytown-setup.ts

```diff
@@ -61,9 +62,45 @@ async function copyPartytownStaticFiles(
   await copyLibFiles(partytownLibDir)
 }
 
+async function verifyRootLayoutImport(appDir: string) {
+  const possibleLayoutFiles = [
+    'layout.ts',
+    'layout.tsx',
+    'layout.js',
+    'layout.jsx',
+  ]
+  let rootLayoutPath: string | undefined
+  let rootLayoutContent: string | undefined
+
+  for (const file of possibleLayoutFiles) {
+    const filePath = path.join(appDir, file)
+    if (await fileExists(filePath, FileType.File)) {
+      rootLayoutPath = filePath
+      rootLayoutContent = await readFile(filePath, 'utf8')
+      break
+    }
+  }
+
+  if (!rootLayoutPath || !rootLayoutContent) return
+
+  const hasPartytownComponent = /<Partytown/i.test(rootLayoutContent)
```

Comment:
This regex would also catch a comment saying `// we're not using <Partytown /> here`, can you make sure the comments are excluded?